lib/spree_one_page_stock_management/factories.rb

```diff
@@ -3,4 +3,16 @@
   #
   # Example adding this to your spec_helper will load these Factories for use:
   # require 'spree_one_page_stock_management/factories'
+  factory :stock_updater, class: Spree::StockUpdater do
+    data_file { File.new(Rails.root + 'lib/test.csv') }
```

Comment:
hi, i think you forgot to include this and the below files in the PR; we want to use gem and it would be great if you included this so the tests pass

thanks!

Reply:
Hi @sent-hil 

All the test files have been added. Thanks for notifying.